src/com/github/ustc_zzzz/virtualchest/VirtualChestPlugin.java

```diff
@@ -61,7 +63,10 @@
  * @author ustc_zzzz
  */
 @Plugin(name = "VirtualChest", id = VirtualChestPlugin.PLUGIN_ID,
-        dependencies = {@Dependency(id = "spongeapi"), @Dependency(id = "placeholderapi", version = "[4.0,)")},
```

Comment:
Do not remove the dependency of SpongeAPI. A new dependency with an API version will be added when the plugin is built and `mcmod.info` is generated if the dependency is not explicitly declared. VC is supposed to support both API 5 and API 7 and the version number of SpongeAPI is not expected.